doc/content/getting-started/migrating/migrating-from-v2/_index.md

```diff
@@ -0,0 +1,90 @@
+
+## Prerequisites
+
+1. User account in {{% ttnv2 %}}.
+2. End device(s) and gateway(s) connected to {{% ttnv2 %}} network.
+3. User account in {{% tts %}}.
+
+{{< note >}} We **highly recommend** using {{% tts %}} version `3.12.0` or higher. Some of the features (like session migration) from this guide might not be available for prior versions. {{</ note >}}
+
+## Packet Broker Role in Migration Process
+
+Before we start with explaining the process of migration, it is important to understand the role of the [Packet Broker]({{< ref "/reference/packet-broker" >}}) in it.
+
+End devices connected to **The Things Network V2** can be migrated to **{{% tts %}} Cloud** and **{{% tts %}} Community Edition** without migrating a gateway, thanks to their default connection to Packet Broker. **The Things Industries V2 (SaaS)** deployments can also be connected to Packet Broker on customer request by contacting [our support](mailto:support@thethingsindustries.com), while **{{% tts %}} Enterprise** or **{{% tts %}} Open Source** can even be [configured]({{< ref "/reference/packet-broker/connect" >}}) by a user to connect to Packet Broker.
+
+If you are using deployments connected to Packet Broker and your gateway is still connected to {{% ttnv2 %}}, the traffic from your end device can be routed to {{% tts %}} (and reverse) after you successfully migrate only your end device. In order for this traffic to be properly routed by the Packet Broker, there are some requirements your devices need to fulfill:
+
+- {{% ttnv2 %}} **DevAddr** of your end device needs to be routable by the Packet Broker.
+    
+    OTAA devices migrated without an active session will acquire a new **DevAddr** from {{% tts %}} during the join procedure. This **DevAddr** will always be routable by the Packet Broker.
+    
+    When registering ABP devices on {{% tts %}}, it is possible to auto-generate new **DevAddr**, that can be properly routed by the Packet Broker. ABP devices can be re-programmed to use this new **DevAddr**. This implies breaking their existing session on {{% ttnv2 %}}, i.e. migrating them to {{% tts %}} without their active session, but this way Packet Broker will always be able to route their traffic to {{% tts %}}.
+
+    Migrating OTAA and ABP devices with their active sessions implies keeping their **DevAddr** values from {{% ttnv2 %}}. Packet Broker will be able to route traffic for these **DevAddr** values only if these devices were registered on **The Things Industries V2 (SaaS)** and you are migrating them to **{{% tts %}} Cloud**. If you are not using these deployments and you still want your traffic to be routed by the Packet Broker, end devices can only be migrated without their active sessions.
+<br>
+</br>
```

Comment:
Is this needed?

Reply:
I put this because otherwise the text gets too close and violates readability a bit. 

Without it:
![Screen Shot 2021-04-21 at 16 11 13](https://user-images.githubusercontent.com/65215579/115410802-3dbf3c80-a1f3-11eb-9bbc-4f644b9f57fc.png)

With it:
![Screen Shot 2021-04-21 at 16 11 05](https://user-images.githubusercontent.com/65215579/115410830-431c8700-a1f3-11eb-80d6-c3b0e2008bb9.png)